### PR TITLE
Hotfix: PHP 7 warnings on binding internal classes

### PR DESCRIPTION
**Pull request by Ocramius**

### CODE REVIEW

File: tests/language-feature-scripts/init.php [hunk 1]
```diff
@@ -7,4 +7,13 @@
 
 $configuration = new Configuration();
 
-$configuration->setGeneratorStrategy(new EvaluatingGeneratorStrategy());
+$locator = new \ProxyManager\FileLocator\FileLocator(__DIR__ . '/../../tmp');
```
Comment (Ocramius) [inline tests/language-feature-scripts/init.php]: Note: to be reverted

**Comment by malukenho** [comment]
:+1: Looks good for me

File: tests/language-feature-scripts/init.php [hunk 2]
```diff
@@ -7,4 +7,9 @@
 
 $configuration = new Configuration();
 
-$configuration->setGeneratorStrategy(new EvaluatingGeneratorStrategy());
+$locator = new \ProxyManager\FileLocator\FileLocator(sys_get_temp_dir());
```
Comment (Ocramius) [inline tests/language-feature-scripts/init.php]: Will revert

**Comment by malukenho** [comment]
@Ocramius you're fast \o/

**Pull request merged**